src/omero/plugins/duplicate.py

```diff
@@ -61,6 +72,38 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Duplicate
 
+    def _pre_objects(self, parser):
+        parser.add_argument(
+            "--duplicate-classes",
+            help=("Modifies the given option by specifying kinds of object to "
```

Comment:
I could see dropping "Modifies the given option by". Perhaps, "Specify kind of object to duplicate"

Reply:
These are phrased to match the similar options in `cli.py` so I'll open an issue suggesting a wider review.